### PR TITLE
Get user by id

### PR DESCRIPTION
**Pull request by jaanmeld**

### CODE REVIEW

File: src/api/users/users.model.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    type: String,
+    required: true,
+  },
+  isAdmin: {
+    type: Boolean,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
```
Comment (Ahuanmarca) [inline src/api/users/users.model.js:19]: I think we should have 'unique' restriction in the email field. What do you think?

**Review by Ahuanmarca — CHANGES_REQUESTED** [review]
Please add your route to router.js in the /api directory.

File: src/api/users/users.router.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Router from 'express';
+import * as usersController from './users.controller.js';
+
+const router = Router();
+
+router.get('/id/:id', usersController.getById);
```
Comment (andreamir) [inline src/api/users/users.router.js]: In the **gameTitle.getById** route, the current implementation is:

`router.get('/:id', gameTitlesController.getById);`

We should aim to maintain a consistent approach for all routes.

**Review by andreamir — CHANGES_REQUESTED** [review]

**Review by nuinn — APPROVED** [review]

**Review by andreamir — CHANGES_REQUESTED** [review]
You have conflicts to resolve before merging into the 'develop' branch.

**Review by Ahuanmarca — APPROVED** [review]

**Review by andreamir — CHANGES_REQUESTED** [review]
You have conflicts to resolve before merging into the 'develop' branch.

**Review by andreamir — APPROVED** [review]

**Review by JonaTheBridge — APPROVED** [review]

**Pull request merged**